sites/docs/src/content/docs/tutorials/contributing_to_nf-core/contributing_to_pipelines.md

```diff
@@ -80,6 +81,20 @@ Only in the unlikely and regretful event of a release happening with a bug.
 - Fix the bug, and bump version (X.Y.Z+1).
 - A PR should be made on `master` from patch to directly resolve this particular bug.
 
+## PR review guidelines
+
+- Authors/Reviewers can request help for review at nf-core Slack `#request-review` channel.
+- Anyone can merge after one positive review and with no obvious open questions and, merge ASAP so as not to leave approved PRs hanging.
+- Exception in merges to master for releases (two reviews).
+- Exception in pseudo PR review for first release (needs review from core or maintainer team).
```

Comment:
Link to docs for first release

Reply:
sorry, I cannot find exactly this? I found pipeline release but not first release

Reply:
https://nf-co.re/docs/tutorials/adding_a_pipeline/first_release